src/test/java/uk/gov/companieshouse/api/testdata/controller/TestDataControllerTest.java

```diff
@@ -450,21 +451,34 @@ void deleteAppealBadRequest() throws Exception {
 
     @Test
     void createCertificateSuccess() throws Exception {
-        CertificatesData certificate = new CertificatesData(
-                "CRT-834723-192847", "2025-04-14T12:00:00Z", "2025-04-14T12:00:00Z"
+        CertificatesData.CertificateEntry entry1 = new CertificatesData.CertificateEntry(
+            "CRT-834723-192847", "2025-04-14T12:00:00Z", "2025-04-14T12:00:00Z"
         );
+        CertificatesData.CertificateEntry entry2 = new CertificatesData.CertificateEntry(
+            "CRT-912834-238472", "2025-04-14T12:05:00Z", "2025-04-14T12:05:00Z"
+        );
+
+        // Use LinkedList to support getFirst()
+        List<CertificatesData.CertificateEntry> entries = new java.util.LinkedList<>(List.of(entry1, entry2));
```

Comment:
Is there a reason to use the fully-qualified class name here?

Also `getFirst()` belongs to the `List` interface not only `LinkedList` as you're stating that the datatype is `List` if it did only belong to `LinkedList` you wouldn't be able to use it later`

Obviously that doesn't technically mean it's implemented properly but with `List.of(entry1, entry2)` that outputs a `List12` which does implement `getFirst()`

I guess all this is to say you can easily remove the `LinkedList` part and just do this.

```suggestion
        List<CertificatesData.CertificateEntry> entries = List.of(entry1, entry2);
```